meta-balena-common/recipes-core/initrdscripts/files/rootfs

```diff
@@ -32,8 +32,9 @@ rootfs_run() {
             fi
 
             if [ "`echo ${bootparam_root} | cut -c1-6`" = "LABEL=" ]; then
-                root_uuid=`echo $bootparam_root | cut -c7-`
-                bootparam_root="/dev/disk/by-label/$root_uuid"
+                root_label=`echo $bootparam_root | cut -c7-`
+                bootparam_root="$(get_state_path_from_label "$root_label")"
+		info "Using bootparam_root $bootparam_root for $root_label"
```

Comment:
tab/spacebar mixup.

Reply:
Commit message says
"we first try to use the by-state udev link and fallback to using the rootfs label if unavailable."

I don't see the fallback. what am I missing?

Reply:
Sorry for the tabs, need to fix my vim formatting.

The comment refers to the following change:
```
             if [ "`echo ${bootparam_root} | cut -c1-6`" = "LABEL=" ]; then
-                root_uuid=`echo $bootparam_root | cut -c7-`
-                bootparam_root="/dev/disk/by-label/$root_uuid"
+                root_label=`echo $bootparam_root | cut -c7-`
+                bootparam_root="$(get_state_path_from_label "$root_label")"
```
So when a LABEL is passed in the kernel cmd line, the get_state_from_path will try to use the udev state symlink first, and if that does not exist, will fallback to using the filesystem label.
Let me know if it can be better expressed.